.env

```diff
@@ -16,3 +16,7 @@ REACT_APP_HSDS_DOMAIN=
 
 # Rendering performance profiling (local development only)
 REACT_APP_PROFILING_ENABLED=false
+
+# Logging limit of React Testing Library
+# https://testing-library.com/docs/dom-testing-library/api-helpers/#debugging
+DEBUG_PRINT_LIMIT=
```

Comment:
When Testing Library doesn't find an element, it logs a whole bunch of DOM elements, which makes it difficult to see which test has failed. Set `DEBUG_PRINT_LIMIT=0` in `.env.local` to remove these logs.